src/services/side-panels/side-panels-manager.ts

```diff
@@ -111,6 +111,18 @@ export class SidePanelsManager {
    * @ignore
    */
   public reset(): void {
+    this.logger.debug('reset');
```

Comment:
remove if not needed

Reply:
it is needed, every service.reset is called on change media see [here](https://github.com/kaltura/kaltura-player-js/blob/master/src/common/service-provider.js#L19)